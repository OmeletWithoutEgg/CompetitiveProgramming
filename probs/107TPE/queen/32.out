969.00 5411839.00
