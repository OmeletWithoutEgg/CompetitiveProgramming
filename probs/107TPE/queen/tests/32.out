-32770979.00 -89856177.00
